<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpFit</class>
 <widget class="QWidget" name="JumpFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblSample">
       <property name="text">
        <string>Sample: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="loadLabelText" stdset="0">
        <string>Plot</string>
       </property>
       <property name="workspaceSuffixes" stdset="0">
        <stringlist>
         <string>_Result</string>
        </stringlist>
       </property>
       <property name="fileBrowserSuffixes" stdset="0">
        <stringlist>
         <string>_Result.nxs</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cbWidth"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbFunction">
       <item>
        <property name="text">
         <string>Chudley-Elliott</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hall-Ross</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fick Diffusion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teixeira Water</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Width: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFunc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fit Function: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="treeSpace"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotArea">
       <item>
        <layout class="QVBoxLayout" name="plotSpace"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="plotLegend">
         <item>
          <spacer name="legendSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="legendRaw">
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="legendFit">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="legendDiff">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>Diff</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_44">
     <item>
      <widget class="QGroupBox" name="gbOutput">
       <property name="title">
        <string>Output Options</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="chkVerbose">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Verbose</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPlot">
          <property name="text">
           <string>Plot Result</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSave">
          <property name="text">
           <string>Save Result</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
